use-cases/extras/3/extras-3.md

```diff
@@ -1,17 +1,17 @@
 ### Use case #3
 
-Name: Users have asked for a way to "unsubscribe" promise listeners ("unthen") - Kyle Simpson opened an issue about this
+Name: Users have asked for a way to "unsubscribe" promise listeners ("unthen", "uncatch", "unfinally") - Kyle Simpson opened an issue about this
 
 Peter is writing code with promises and is surprised by the fact he cannot unsubscribe from a promise once he has subscribed for it.
 
 ### What happens ?
 
-It is impossible to unsubscribe from promises at the moment and it is impossible to write code that does this with a promise subclass that operates correctly.
+It is impossible to unsubscribe from promises at the moment and it is [inefficient/ugly](https://gist.github.com/getify/1173cac45d15fc4ff0a880f32fd598ab#file-2-js-L93-L125) to write code that does this with a promise subclass that operates correctly.
```

Comment:
Would it be OK if I remove the word "ugly" here and replace it with something like "inelegantly"? Feel free to disagree

Reply:
"inelegant" would be fine if you prefer. :)